### PR TITLE
feat(route):add College of Pharmacy to zcmu

### PR DESCRIPTION
**Pull request by i-CraftY**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/zcmu/jwc/0
/zcmu/jwc/1
/zcmu/jwc/2
/zcmu/jwc/3
/zcmu/jwc/4
/zcmu/jwc/5
/zcmu/yxy/0
/zcmu/yxy/1
/zcmu/yxy/2
/zcmu/yxy/3
/zcmu/yxy/4
/zcmu/yxy/5
/zcmu/yxy/6
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note
improve route of /zcmu/jwc and the docs

### CODE REVIEW

File: lib/v2/zcmu/radar.js [hunk 1]
```diff
@@ -1,10 +1,86 @@
 module.exports = {
     'zcmu.edu.cn': {
         _name: '浙江中医药大学',
-        '.': [
+        'jwc.zcmu': [
```
[the remark this answers is not in the document]
Comment (CCraftY) [reply lib/v2/zcmu/radar.js]: updated,the FQDN is expected to be `jwc.zcmu.edu.cn`.

**Comment by CCraftY** [comment]
fixed a problem of link

File: docs/university.md [hunk 2]
```diff
@@ -3253,9 +3253,19 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
...
-| 1    | 2    | 3    | 4    | 5    |
+|教务管理| 成绩管理 | 学籍管理 | 考试管理 | 选课管理 | 排课管理 |
+|---| ---- | ---- | ---- | ---- | ---- |
+|0| 1    | 2    | 3    | 4    | 5    |
+
+</Route>
+
+### 药学院
+
+<Route author="CCraftY" example="/zcmu/yxy/0" path="/zcmu/yxy/:type?" :paramsDecs="['必选，模块id']">
```
Comment (TonyRL) [inline docs/university.md:3264]:
```suggestion
<Route author="CCraftY" example="/zcmu/yxy/0" path="/zcmu/yxy/:type?" :paramsDecs="['模块id']">
```

File: lib/v2/zcmu/yxy/index.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                title: item.find('a').text(),
+                link: `https://yxy.zcmu.edu.cn/${item.find('a').attr('href')}`,
+                pubDate: parseDate(item.find('span').text().trim()),
+            };
+        })
+        .get();
+
+    ctx.state.data = {
+        title: map.get(type).title,
+        link: `${host}/${id}`,
```
Comment (TonyRL) [inline lib/v2/zcmu/yxy/index.js:39]:
```suggestion
        link: `${host}${id}`,
```

**Pull request merged**